### PR TITLE
[IMP] hr_holidays: Allow accruals based on worked hours

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays_attendance/models/hr_leave_accrual_plan_level.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class AccrualPlanLevel(models.Model):
+    _inherit = "hr.leave.accrual.level"
+
+    frequency_hourly_source = fields.Selection(
+        selection=[
+            ('calendar', 'Calendar'),
+            ('attendance', 'Attendances')
+        ],
+        default='calendar',
+        compute='_compute_frequency_hourly_source',
```
Comment (Bertrand2) [inline addons/hr_holidays_attendance/models/hr_leave_accrual_plan_level.py:16]: As we want to ensure that `frequency_hourly_source` is set to calendar if accrual is set at start, shouldn't we have an inverse setting `accrued_gain_time` to 'end' if we set attendance?

**Comment by tivisse** [comment]
@robodoo r+ rebase-ff

**Comment by tivisse** [comment]
> @robodoo r-
> 
> This seems to be broken: https://runbot.odoo.com/runbot/build/51548501
> 
> Not entirely sure, but the PR talks about accrual, and 3 different stagings broke the same way when it was present.

Thanks for the heads up, I'm gonna take a look at it. :)

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**